src/forwarder.ts

```diff
@@ -63,6 +79,43 @@ async function validateForwarderArn(aws: Aws, functionArn: CloudFormationObjectA
   }
 }
 
+export async function addExecutionLogGroupsAndSubscriptions(
+  service: Service,
+  aws: Aws,
+  functionArn: CloudFormationObjectArn | string,
+) {
+  const extendedProvider = (service.provider as any)?.logs;
```

Comment:
Why is this called `extendedProvider`?

Reply:
This is because the Serverless class from the types module doesn't include the logs parameter in the provider section. We needed to create a union and extend it with logs, which has a specific type we created.

Reply:
Hmm, but isn't the "extended provider" here JUST the logs config section?